src/Idris/Doc/String.idr

```diff
@@ -454,7 +466,8 @@ getDocsForName fc n config
                   in annotate DocStringBody
                      (concatWith (\l, r => l <+> hardline <+> r) docs)
                      <$ guard (not (null docs))
-             pure (vcat (docDecl :: docBody))
+             let maybeDocDecl = [docDecl | showType]
+             pure . vcat . catMaybes $ maybeDocDecl :: (map Just $ docBody)
```

Comment:
Should the `:doc` output also include a `DEPRECATED` string?

Reply:
Yeah, that seems smart. Will do.